### PR TITLE
[CAS-1072] Reorganize client docs

### PR DESCRIPTION
**Pull request by adasiewiczr**
### 🎯 Goal

Reorganize client docs

### 🧪 Testing

1. Run `npx stream-chat-docusaurus -i -s`
2. Check `Client` section

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (CMS, cookbooks, tutorial)
- [x] Reviewers added

### CODE REVIEW

File: docusaurus/docs/Android/02-client/06-push-notifications.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # Push Notifications
 
+Besides [Events](./04-events.md), push notifications are another way to stay up to date with changes to the chat.
+The user will receive a push notification if he watches the channel and doesn't have an open socket connection.
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/06-push-notifications.md:4]:
```suggestion
The user will receive a push notification if they're watching a channel but don't have an open socket connection.
```

File: docusaurus/docs/Android/02-client/04-events.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Events
 
+Events allow the client to stay up to date with changes to the chat. Examples are a new message, a user's image that updated, a reaction, or a member joining the channel.
+Events are only received when SDK is connected to the socket and the application is in the foreground.
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/04-events.md:4]:
```suggestion
Events are only received when the client is connected to the socket and the application is in the foreground.
```

Plus we could also link to push notifications here, as that's what you get if the socket connection is not active

File: docusaurus/docs/Android/02-client/04-events.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Events
 
+Events allow the client to stay up to date with changes to the chat. Examples are a new message, a user's image that updated, a reaction, or a member joining the channel.
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/04-events.md:3]:
```suggestion
Events allow the client to stay up to date with changes to the chat. For example, you get events when a new message was sent, a user's image was updated, a reaction was added, or a member joined the channel.
```

File: docusaurus/docs/Android/02-client/03-messages.md [hunk 4]
```diff
@@ -409,13 +376,8 @@ client.searchMessages(
 }
 ```
 
-<!-- TODO: Do these MongoDB mentions here make any sense at all? -->
-
 Pagination works via the standard `limit` and `offset` parameters. The first argument, `filter`, uses a MongoDB style query expression.
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/03-messages.md:379]: This still talks about MongoDB

File: docusaurus/docs/Android/02-client/03-messages.md [hunk 5]
```diff
@@ -254,34 +251,23 @@ channelClient.deleteReaction(
...
- 
-// Get 10 reactions after particular reaction 
-channelClient.getReactions( 
-    messageId = "message-id", 
-    firstReactionId = "reaction-id", 
-    limit = 10, 
+).enqueue { /* ... */ }
+```
+
+You can also get reaction added after particular reaction:
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/03-messages.md:263]:
```suggestion
You can also get the reactions added after a particular reaction:
```

File: docusaurus/docs/Android/02-client/02-channels.md [hunk 6]
```diff
@@ -1,6 +1,8 @@
 # Channels
 
-<!-- TODO: Add brief intro about what channels are -->
+Channel allows to make conversation between two or more chat users. It can be identified by `id` and `type`, or `cid` which is a combination of `type:id`
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/02-channels.md:3]:
```suggestion
Channels are where conversations take place between two or more chat users. They contain a list of messages and have a list of the member users that are participating in the conversation.

A channel is identified by its `type` and `id`. Some APIs use a `cid` to identify a channel with a single string - this is the combination of the two pieces of information, as `type:id`.
```

Comment (adasiewiczr) [reply docusaurus/docs/Android/02-client/02-channels.md:3]: Sounds much better. Thanks!

File: docusaurus/docs/Android/02-client/03-messages.md (hunk 4 above)
Comment (adasiewiczr) [reply docusaurus/docs/Android/02-client/03-messages.md:379]: Nice catch!

**Review by nikomancy — APPROVED** [review]
Looks good.

**Pull request merged**